File: FoxChat-java/foxChat-pojo/src/main/java/com/bedfox/pojo/domain/LlmUser.java
package com.bedfox.pojo.domain;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.io.Serializable;

/**
 * 
 * @TableName llm_user
 */
@TableName(value ="llm_user")
@Data
public class LlmUser implements Serializable {
    /**
     * ID
     */
    @TableId
    private String id;

    /**
     * 用户ID
     */
    private String userId;

    /**
     * 模型昵称
     */
    private String llmName;

    /**
     * 模型头像
     */
    private String faceImage;

    /**
     * 初始化记忆
     */
    private String memoryContent;

    @TableField(exist = false)
    private static final long serialVersionUID = 1L;
}

File: FoxChatRAG-python/app/models/llm_user.py
from sqlalchemy import VARCHAR, Column
from sqlalchemy.ext.declarative import declarative_base

Base = declarative_base()

class LlmUser(Base):
    __tablename__ = "llm_user"
    id = Column(VARCHAR, primary_key=True)
    user_id = Column(VARCHAR)
    llm_name = Column(VARCHAR)
    face_image = Column(VARCHAR)
    memory_content = Column(VARCHAR)
